openapi/resource-server.yaml

```diff
@@ -291,27 +291,60 @@ paths:
                   quoteId: 'https://ilp.rafiki.money/quotes/ab03296b-0c8b-4776-b94e-7ee27d868d4d'
                   metadata:
                     externalRef: INV2022-02-0137
+              Create an outgoing payment based on an incoming payment:
+                value:
+                  walletAddress: 'https://ilp.rafiki.money/alice/'
+                  incomingPaymentId: 'https://ilp.rafiki.money/incoming-payments/8d4e4776-2e55-4e5a-bcbe-8348ed1e86de'
```

Comment:
can we just call that `incomingPayment` without the `Id`-part? I think that is confusing.

Reply:
I have always found this confusing as well (I would assume id is the uuid) but I think it's the most consistent, and I think `incomingPayment` would also be confusing. 

Looking at the rest of the spec ids are generally referring to urls. such as `id` on the `incomingPayment` object and `quoteId` on the outgoing payment object.

Personally I would opt for `incomnigPaymentUrl` or `incomnigPaymentUri` as `incomingPayment` seems like it would be the full object (which itself has an id which is a url). I guess a counterpoint is the `walletAddress` which i see is used for both a url and an object. Although I would argue that shouldn't be emulated.

Reply:
still mulling this over...

Reply:
Good point, `...Url` or `...Uri` is probably the best way to name it. It's a bit breaking change though... Maybe we keep it at `...Id` for now and bring it up during the Open Payments call

Reply:
I think that  `incomingPayment` without any specifiers like `id` in the name is pretty open to interpretation, so long as we're consistent. [Stripe mixes it up between a fully expanded object or just an id between fields in the same object](https://docs.stripe.com/api/charges/object), for example. So in our case I think we could have it be a URL if we wanted.

Reply:
:+1: for `-Url`

Reply:
In our planning call we decided on `incomingPayment` because its clearer and no less consistent than `incomingPaymentId`. Doing `incomingPaymentUrl` would be a new pattern and would require substantial renaming. Will discuss in the Open Payments call and tackle that in a separate issue, if any.